test_helpers/shared_feature_context.go

```diff
@@ -115,6 +130,16 @@ func SharedFeatureContext(s *godog.Suite) {
 		return childCmdPlus.Start()
 	})
 
+	s.Step(`^starting "([^"]*)" in the "([^"]*)" directory$`, func(command, dirName string) error {
+		commandWords, err := util.ParseCommand(command)
+		if err != nil {
+			return err
+		}
+		childCmdPlus = execplus.NewCmdPlus(commandWords...)
+		appDir = path.Join(appDir, dirName)
```

Comment:
Do we need to update the appDir variable here? I think we can pass `path.Join(appDir, dirName)` directly into    `childCmdPlus.SetDir`. Again, I don't like that its being set to something thats not an application directory.